client/scripts/GLOBAL.as

```diff
@@ -572,114 +572,114 @@ package
-                  "r1": 2160,
-                  "r2": 2160,
-                  "r3": 0,
-                  "r4": 0,
-                  "time": 300,
-                  "re": [[14, 1, 1]]
-               }, {
-                  "r1": 8640,
-                  "r2": 8640,
-                  "r3": 0,
-                  "r4": 0,
-                  "time": 4500,
-                  "re": [[14, 1, 3], [8, 1, 1]]
-               }, {
-                  "r1": 34560,
-                  "r2": 34560,
-                  "r3": 0,
-                  "r4": 0,
-                  "time": 10800,
-                  "re": [[14, 1, 4], [8, 1, 1]]
-               }, {
-                  "r1": 138240,
-                  "r2": 138240,
-                  "r3": 0,
-                  "r4": 0,
-                  "time": 28800,
-                  "re": [[14, 1, 5], [8, 1, 1]]
-               }, {
-                  "r1": 552960,
-                  "r2": 552960,
-                  "r3": 0,
-                  "r4": 0,
-                  "time": 72000,
-                  "re": [[14, 1, 6], [8, 1, 1]]
-               }, {
-                  "r1": 2211840,
-                  "r2": 2211840,
-                  "r3": 0,
-                  "r4": 0,
-                  "time": 144000,
-                  "re": [[14, 1, 6], [8, 1, 1]]
-               }];
-         _buildingProps[14].capacity = [200, 260, 320, 380, 450, 540];
-         _buildingProps[14].hp = [4000, 14000, 25000, 43000, 75000, 130000];
-         _buildingProps[14].repairTime = [100, 200, 300, 400, 500, 600];
-         _buildingProps[21].capacity = [380, 450, 540, 660, 800];
-         _buildingProps[4].costs = [ {
-                  "r1": 1000,
-                  "r2": 1000,
-                  "r3": 500,
-                  "r4": 0,
-                  "time": 900,
-                  "re": [[14, 1, 1]]
-               }, {
-                  "r1": 64300,
-                  "r2": 64300,
-                  "r3": 32150,
-                  "r4": 0,
-                  "time": 10800,
-                  "re": [[14, 1, 3], [11, 1, 1]]
-               }, {
-                  "r1": 283600,
-                  "r2": 283600,
-                  "r3": 141800,
-                  "r4": 0,
-                  "time": 32400,
-                  "re": [[14, 1, 4], [11, 1, 1]]
-               }, {
-                  "r1": 1247840,
-                  "r2": 1247840,
-                  "r3": 623920,
-                  "r4": 0,
-                  "time": 97200,
-                  "re": [[14, 1, 4], [11, 1, 1]]
-               }, {
-                  "r1": 1247840,
-                  "r2": 1247840,
-                  "r3": 623920,
-                  "r4": 0,
-                  "time": 97200,
-                  "re": [[14, 1, 4], [11, 1, 1]]
-               }];
-         _buildingProps[4].hp = [4000, 8000, 16000, 28000, 28000];
-         _buildingProps[4].capacity = [500, 1000, 1750, 2250, 3000, 4000, 4000];
+         _buildingProps[8].costs = [{
```

Comment:
Looks good! Let's just be careful with formatting changes.

Reply:
Sorry for the formatting mess. I was flipflopping between VSCode and JPEXS and never sat down to clean it up.